src/pyload/core/managers/thread_manager.py

```diff
@@ -201,9 +201,8 @@ def try_reconnect(self):
         self.pyload.log.debug(f"Old IP: {ip}")
 
         try:
-            reconn = subprocess.Popen(
-                reconnect_script, bufsize=-1, shell=True
-            )  #: , stdout=subprocess.PIPE)
+            #Using shell in subprocess can be unsecure
+            reconn = subprocess.Popen(reconnect_script, bufsize=-1, shell=False)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5106881)